functional/MBCS_Tests/datetime/build.xml

```diff
@@ -27,13 +27,16 @@ limitations under the License.
 	<!--Properties for this particular build-->
 	<property name="src" location="./src" />
 	<property name="build" location="./bin" />
+	<property name="junit4.jar" location="${LIB_DIR}/junit4.jar" />
```

Comment:
Since line 49 is using `${LIB_DIR}/junit4.jar" directly, do we still need to set this property here? Same for other two files.

Reply:
Removed the line `<property name="junit4.jar" location="${LIB_DIR}/junit4.jar" />`. 
Grinder link:
**TARGET: testList TESTLIST=MBCS_Tests_datetime_0,MBCS_Tests_datetime_formatter_0**
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43550/
**special.functional**
https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/43554/console